### ISSUE TITLE
Need @3x assets - iPhone 6+ Support

### ISSUE THREAD

**Issue by davecom** [cjwirth/RichEditorView#1]

**Comment by cjwirth** [comment]
I agree, this is definitely something that needs addressing. I will try to find/make some this weekend. 

In the meantime, you can add in your own items with custom images, and just copy their actions.

**Comment by cjwirth** [comment]
Gave this a shot in Gimp and Inkscape about 3 times. They end up looking terrible, so if anyone wants to help out, that would be great.

**Comment by jesiegel1** [comment]
@cjwirth I can take care of this - any chance you have a single file with all of the assets?  Would make it easier, otherwise I can just create @3x assets from each asset in the library.

**Comment by cjwirth** [comment]
I don't, no. I took the icons from https://github.com/nnhubbard/ZSSRichTextEditor . This project was very inspired by that project, but maybe like a "light" version, because it's not a View Controller, doesn't have color pickers, etc...

I did try making some out of scratch, but failed horribly, and here we are 😄

**Comment by terenzeyuen** [comment]
Maybe change them all to use vector PDF?

**Comment by cjwirth** [comment]
Yeah, that would work. The original assets I got were unfortunately PNGs though. I'm no good with graphics, but if you could get some, that'd be cool.

**Comment by terenzeyuen** [comment]
Im sourcing some for my custom toolbar so should be able to provide some.
